system/lib/libc/musl/src/conf/sysconf.c

```diff
@@ -161,7 +161,7 @@ long sysconf(int name)
 		[_SC_TRACE_NAME_MAX] = -1,
 		[_SC_TRACE_SYS_MAX] = -1,
 		[_SC_TRACE_USER_EVENT_MAX] = -1,
-		[_SC_XOPEN_STREAMS] = JT_ZERO,
+		[_SC_XOPEN_STREAMS] = -1, // XXX EMSCRIPTEN replace JT_ZERO,
```

Comment:
I don't really like all these local changes that don't seem to serve any functional purpose other than agreeing with our JS implementation..  can we make the test more flexible instead?

Reply:
I think the benefit of avoiding user-noticeable changes is worth it. If users get different sysconf results when changing between standalone and normal mode that can be annoying - some user will be debugging something, and end up chasing a red herring because of a difference like this.

I'd prefer us to have the same in both modes. I don't have a strong feeling between the two, but the old values have the benefit of not having any complaints (AFAIK) and that keeping them avoids users seeing a difference when they update.